torch/nn/modules/module.py

```diff
@@ -419,8 +424,13 @@ def to(self, *args, **kwargs):
                 raise TypeError('nn.Module.to only accepts floating point '
                                 'dtypes, but got desired dtype={}'.format(dtype))
 
+        convert_to_format = kwargs.get('memory_format', None)
```

Comment:
is there a reason you didn't modify _parse_to?  It seems nice to keep the parsing logic in one place.